packages/core/core/package.json

```diff
@@ -31,6 +31,20 @@
       "require": "./dist/index.js",
       "default": "./dist/index.js"
     },
+    "./strapi": {
+      "types": "./dist/facades/strapi.d.ts",
+      "source": "./src/facades/strapi.ts",
+      "import": "./dist/facades/strapi.mjs",
+      "require": "./dist/facades/strapi.js",
+      "default": "./dist/facades/strapi.js"
+    },
+    "./config": {
+      "types": "./dist/facades/config.d.ts",
+      "source": "./src/facades/config.ts",
```

Comment:
where does the config file comes from?

Reply:
I forgot to delete it I was experimenting with more facades ;)